test/functional/eval/null_spec.lua

```diff
@@ -60,7 +60,7 @@ describe('NULL', function()
     -- FIXME should be accepted by writefile(), return {0, {}}
     null_expr_test('is accepted as an empty list by writefile()',
                         ('[writefile(L, "%s"), readfile("%s")]'):format(tmpfname, tmpfname),
-                        'E484: Can\'t open file ' .. tmpfname, {0, {}})
+                        'E484: Cannot open file ' .. tmpfname, {0, {}})
```

Comment:
I'll defer to you two on that, and leave it in your hands.  I'll get on with #6777 and #6795.